### DISCUSSION TITLE
Errore 1004 e info da principiante

### DISCUSSION THREAD

**Discussion by dianasimone** [indomus/forum#1617] [HUB: Home Assistant]
Buongiorno a tutti, sono nuovo del forum e anche di home assistant. Premessa, non sono uno smanettone e non ci acchiappo moltissimo, sono piu' ferrato in impianti elettrici o domotici "classici", con sonoff, shelly ecc... Vorrei installare e utilizzare home assistant al posto di un hub che utilizzo ora, HOMEY, in quanto mia moglie lo ha rotto e la parte zigbee non funziona piu. Ho installato HA su un raspberry 3 e per la prima settimana smanettando funzionava, poi ho tolto corrente per sistemare l'impianto e di punto in bianco mi è comparso la scritta "could not connect to the server. nsurle error domain 1004".
Inoltre volevo chiedere come accedere da smartphone o da altro terminale esterno senza duckdns, visto che ho un ip fisso.
Insultatemi pure visto che sono un novellino!!
Scherzi a parte grazie in anticipo per le risposte.

**Comment by KC-inDomus** [comment]
ciao Simo,
come lo avevi installato? con HassOS?

**Reply by dianasimone** [reply]
Si, esatto. ho incominciato a smanettarci un pochetto. io credo che sia dovuto a qualcosa del router, ma magari dico fesserie. Il bello è che io ho un'ip fisso, non dovrei neanche picchiarmi con duckdns, giusto?

**Reply by indomus** [reply]
vedi qui: https://indomus.it/formazione/gli-imprescindibili-gli-add-on-essenziali-da-installare-su-hassio/#duckdns

**Reply by dianasimone** [reply]
si questo lo avevo visto, ma il primo problema è che mi da l'errore 1004, poi successivamente vorrei sapere come collegarmi da esterno avendo un'ip fisso (quindi in teoria non dovrei usare duckdns, giusto?).
Ma prima comunque devo fare funzionare tutto l'ambaradan, altrimenti non faccio nulla... Devo reinstallare tutto?

**Comment by dianasimone** [comment]
Aggiornamento, il problema error 1004 è sparito, sono riuscito nuovamente ad entrare in home assistant... Non capisco come mai. Quello che non riesco a trovare è come cavolo si puo' entrare da esterno, avendo un ip fisso e non usando duckdns.

**Reply by indomus** [reply]
ti basta puntare all'IP (ovviamente senza https)

**Reply by Scialla** [reply]
Giusto per farti capire: Se hai IP fisso non risolvi tantissimo, accedi ad esempio con:

http://tuoIP:8123

ma così sei in http e hai protezione zero

Se vuoi accedere in https sei costretto ad avere un dominio perché i certificati non vengono rilasciati su base IP ma su fqdn; per fare questo puoi rivolgerti a qualsiasi servizio, gratuito, che offre la possibilità di avere un dominio, tipo duckdns oppure noip o altri, per poi avere i certificati e quindi accederai con:

https://nome.tuo.dominio:8123

al quale sarà collegato il tuo IP statico o dinamico che sia; l'unica differenza e che avendo lo statico non devi aggiornare su duckdns o quel che ti pare l'IP quando cambia.
